src/interface/src/app/services/plan-notes.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+
+import { BackendConstants } from '../backend-constants';
+import { HttpClient } from '@angular/common/http';
+
+export interface Note {
```

Comment:
Actually, how do you feel about adding an ` id: number` here? I realize that might be helpful..